backend/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "backend",
```

Comment:
Instead of backend write score_keeper / score_keeper_server

Reply:
Done